### PR TITLE
NMS-14526: Karaf commands for Pollerd/Collectd scheduler

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://opennms.atlassian.net/browse/NMS-14526

### CODE REVIEW

File: docs/modules/operation/pages/deep-dive/performance-data-collection/shell/collectd-schedule.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+----
+
+* You can alter the output by specifying `--columns/-c` using a comma-seperated list of the columns to be displayed.
++
+[source, karaf]
+----
+opennms:collectd-schedule --services 0 --columns nodeId,readyTimeMs
+----
+
+* You can alter the order of the displayed service schedules by using the option `--option/-o`. To switch to a descending order specify the additional option `--desc/-d`.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/performance-data-collection/shell/collectd-schedule.adoc:29]:
```suggestion
* You can alter the order of the displayed service schedules by using the option `--option/-o`. 
To switch to a descending order specify the additional option `--desc/-d`.
```

File: docs/modules/operation/pages/deep-dive/service-assurance/shell/introduction.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+= Shell Commands
+
+A number of Karaf Shell commands are made available to help administer and diagnose issues related to service assurance.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/service-assurance/shell/introduction.adoc:4]:
```suggestion
A number of Karaf Shell commands are available to help administer and diagnose issues related to service assurance.
```

File: docs/modules/operation/pages/deep-dive/service-assurance/shell/introduction.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+To use the commands, log into the Karaf Shell on your system using:
+
+[source]
+----
+ssh -p 8101 admin@localhost
+----
+
+NOTE: The Karaf shell uses the same credential as the web interface.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/service-assurance/shell/introduction.adoc:13]:
```suggestion
NOTE: The Karaf shell uses the same credentials as the web interface.
```

File: docs/modules/operation/pages/deep-dive/service-assurance/shell/introduction.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+      Users must be associated with the `ADMIN` role to access the shell.
+
+IMPORTANT: In order to keep the session open while executing long-running tasks without any user input add `-o ServerAliveInterval=10` to your ssh command.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/service-assurance/shell/introduction.adoc:16]:
```suggestion
IMPORTANT: To keep the session open while performing long-running tasks without any user input add `-o ServerAliveInterval=10` to your ssh command.
```

File: docs/modules/operation/pages/deep-dive/service-assurance/shell/pollerd-schedule.adoc [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+The `opennms:pollerd-schedule` Karaf Shell command can be used to output details of the Pollerd scheduler.
+
+* List all collected statistics of the Pollerd daemon.
++
+[source, karaf]
+----
+opennms:pollerd-schedule
+----
+
+* Use `--services/-s` to output a number of scheduled services. The option's argument specifies the limit. Use `0` to display all scheduled services.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/service-assurance/shell/pollerd-schedule.adoc:14]:
```suggestion
* Use `--services/-s` to output a number of scheduled services.
The option's argument specifies the limit.
Use `0` to display all scheduled services.
```

File: docs/modules/operation/pages/deep-dive/service-assurance/shell/pollerd-schedule.adoc [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
++
+[source, karaf]
+----
+opennms:pollerd-schedule --services 0
+----
+
+* You can alter the output by specifying `--columns/-c` using a comma-seperated list of the columns to be displayed.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/service-assurance/shell/pollerd-schedule.adoc:21]:
```suggestion
* You can alter the output by specifying `--columns/-c` using a comma-seperated list of the columns to display.
```

File: docs/modules/operation/pages/deep-dive/service-assurance/shell/pollerd-schedule.adoc [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
++
+[source, karaf]
+----
+opennms:pollerd-schedule --services 0 --columns nodeId,readyTimeMs
+----
+
+* You can alter the order of the displayed service schedules by using the option `--option/-o`. To switch to a descending order specify the additional option `--desc/-d`.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/service-assurance/shell/pollerd-schedule.adoc:28]:
```suggestion
* You can alter the order of the displayed service schedules by using the option `--option/-o`.
To switch to a descending order specify the additional option `--desc/-d`.
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
minor changes. thanks!

**Review by Bonrob2 — APPROVED** [review]
LGTM. Thank you!

**Review by RangerRick — APPROVED** [review]

**Pull request merged**